webapp/components/instantiation/NetPyNEInstantiated.js

```diff
@@ -72,7 +72,7 @@ export default class NetPyNEInstantiated extends React.Component {
   };
 
   updateInstances () {
-    if (Instances.network) {
+    if (typeof Instances !== 'undefined' && Instances.network) {
```

Comment:
safer would be to do `window.Instances !== null && window.Instances.network`.

Using `typeof X !== 'undefined'` only checks if the variable is declared. However`Instances = null` would be a declared variable and when you do `Instances.network` it would throw a TypeError because `network` is not an attribute of `null`.